console/src/tasks.rs

```diff
@@ -126,14 +126,15 @@ impl State {
 
             let id = task.id?.id;
             let stats = stats_update.remove(&id)?.into();
-            let task = Task {
+            let mut task = Task {
                 id,
                 id_hex: format!("{:x}", id),
                 fields: task.string_fields,
                 kind,
                 stats,
                 completed_for: 0,
             };
+            task.updated();
```

Comment:
again, i would call this
```suggestion
            task.update();
```